org-clock-csv-tests.el

```diff
@@ -55,6 +55,10 @@
   "Test tasks with headline ancestors, as in issue #3."
   (org-clock-csv-should-match "tests/issue-3.org" "tests/issue-3.csv"))
 
+(ert-deftest test-issue-5 ()
+  "Test file level category."
+  (org-clock-csv-should-match "tests/issue-5.org" "tests/sample.csv"))
```

Comment:
I believe you may have forgotten to check in the corresponding `.csv` file for this test.